runtime/src/main/java/org/corfudb/runtime/view/ManagementView.java

```diff
@@ -0,0 +1,82 @@
+package org.corfudb.runtime.view;
+
+import java.util.List;
+import java.util.UUID;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+import org.corfudb.protocols.wireprotocol.orchestrator.CreateWorkflowResponse;
+import org.corfudb.protocols.wireprotocol.orchestrator.QueryResponse;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.clients.ManagementClient;
+import org.corfudb.util.Utils;
+
+/**
+ * A view of the Management Service to manage reconfigurations of the Corfu Cluster.
+ *
+ * <p>Created by zlokhandwala on 11/20/17.</p>
+ */
+@Slf4j
+public class ManagementView extends AbstractView {
+
+    private final long layoutRefreshTimeout = 500;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)